src/System.IO.FileSystem/tests/FileStream/LockUnlock.cs

```diff
@@ -90,16 +86,16 @@ public void NonOverlappingRegions_Success(long fileLength, long firstPosition, l
             }
         }
 
-        [ActiveIssue(5964, TestPlatforms.AnyUnix)]
         [Theory]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Good.  It might also be worth codifying this in a unix-only test, to show that, for example, if you have two FileStreams for the same file, lock on one and unlock on the other, that another process is then able to access the region as well.